traitsui/wx/table_editor.py

```diff
@@ -1030,13 +1030,13 @@ def _update_toolbar(self, has_selection):
                         deletable(item) for item in self.selected_items
                     )
 
-                toolbar.search.enabled = toolbar.add.enabled = True
+                toolbar.search.enabled = no_filter
```

Comment:
This is not right.

Reply:
Is it fixed now, or is it still wrong?

Reply:
Still wrong - haven't gotten back to it since looking over yesterday.